src/Logging.jl

```diff
@@ -14,23 +14,25 @@ function Logging.handle_message(logger::AzureSinkLogger, args...; kwargs...)
     task::String = logger.action_chooser(log)
     
     properties=String[]
-    for key in (:level, :_module, :group, :id, :file, :line)
+    # AZP calls it `type` not level`, and `warning` not `warn`:
+    levelmap = Dict(Logging.Error => "error", Logging.Warn => "warn")
```

Comment:
```suggestion
    levelmap = Dict(Logging.Error => "error", Logging.Warn => "warning")
```

Reply:
......hilarious.  I can get it right in the commit message but not in the code?

Reply:
My Japanese TA always chided me for making 'careless mistakes' in my tests.  I guess the legacy continues.